build/detect-secrets.js

```diff
@@ -107,9 +118,13 @@ const detectSecretsScan = (args, stdio) => {
 
 // Ensure that detect-secrets is installed
 try {
-	detectSecrets('--version');
+	// pipe the stdio so we print our own error message if detect-secrets is not installed
+	detectSecrets('--version', stdio = 'pipe', throwIfError = true);
 } catch (error) {
+	printDebug(error);
+	console.error(); // print newline
 	console.error(`${'Error:'.red} detect-secrets is not installed. Install detect-secrets with ${'pip install detect-secrets'.magenta} or ${'brew install detect-secrets'.magenta}.`);
+	console.error(); // print newline
```

Comment:
Not terribly important but you can avoid these extra calls. 

```suggestion
	console.error(`\n${'Error:'.red} detect-secrets is not installed. Install detect-secrets with ${'pip install detect-secrets'.magenta} or ${'brew install detect-secrets'.magenta}.\n`);
```